.storybook/main.cjs

```diff
@@ -102,6 +102,17 @@ module.exports = {
           'url-loader',
         ],
         include: [/images\/.*\.svg$/],
+      },
+      {
+        test: /\.(png|jpe?g|gif|webp)$/i,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              outputPath: 'images',
```

Comment:
```suggestion
              outputPath: '../images',
```

Currently, images are saved in the following path. `assets/js/images`. Changing the value to this will save the images in `assets/images`